src/Controller/HomeController.php

```diff
@@ -1,26 +1,16 @@
 <?php
 
-/**
- * Created by PhpStorm.
- * User: aurelwcs
- * Date: 08/04/19
- * Time: 18:40
- */
-
 namespace App\Controller;
 
 class HomeController extends AbstractController
 {
-    /**
-     * Display home page
-     *
-     * @return string
-     * @throws \Twig\Error\LoaderError
-     * @throws \Twig\Error\RuntimeError
-     * @throws \Twig\Error\SyntaxError
-     */
     public function index()
     {
-        return $this->twig->render('Home/index.html.twig');
+        return $this->twig->render('Home/index.html.twig') . $this->accueil();
+    }
+
+    public function accueil()
```

Comment:
idem, home et pas accueil => Nommage en anglais ;-)

Reply:
mais à la réflexion, tu as déjà une methode index, elle est suffisante, je comprend pas trop pourquoi tu créé accueil en plus, ni pourquoi dans le return de index() tu lui concatène $this->accueil(), à revoir